### PR TITLE
Ronas it/raktaev/update readme

### PR DESCRIPTION
**Pull request by RuslanAktaev**
@astorozhevsky check, please

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -225,3 +236,148 @@ export class AuthEffects {
...
+```
+
+##### ApiConfig
+
+Name | Type | Required | Description
+--- | --- | --- | ---
+`apiUrl` | `string` | Yes | Endpoint that allows you to access an API
+`trailingSlash` | `boolean` | No | The need for trailing slash (`https://api.your-service.com/login/` for example)
+`enableArrayKeys` | `boolean` | No | Enabling array keys for http params
+`fileKeys` | `Array<string>` | 290 | List of the file keys for http params
```
Comment (astorozhevsky) [inline README.md:257]:
```suggestion
`fileKeys` | `Array<string>` | No | List of the file keys for http params

```

File: README.md [hunk 2]
```diff
@@ -225,3 +236,148 @@ export class AuthEffects {
...
+```ts
+CommonAuthModule.forRoot(config: AuthConfig)
+```
+
+##### AuthConfig
+
+Name | Type | Required | Description
+--- | --- | --- | ---
+`unauthorizedRoutes` | `Array<string>` | Yes | Routes that don't need authorization (public routes, e.g. login, registration and forgot password pages)
+`authService` | `Function` | Yes |
```
Comment (astorozhevsky) [inline README.md:287]:
```suggestion
`authService` | `Function` | Yes |  Service that will be used in your app
```

File: README.md [hunk 3]
```diff
@@ -225,3 +236,148 @@ export class AuthEffects {
...
+
+##### AuthConfig
+
+Name | Type | Required | Description
+--- | --- | --- | ---
+`unauthorizedRoutes` | `Array<string>` | Yes | Routes that don't need authorization (public routes, e.g. login, registration and forgot password pages)
+`authService` | `Function` | Yes | 
+`unauthenticatedRoute` | `string` | No | Route to redirect to after logout or when a user is not authenticated to view some page. By default it is set to `/login`
+`authenticatedRoute` | `string` | No | Route to redirect after successful login
+`loginEndpoint` | `string` | No |
```
Comment (astorozhevsky) [inline README.md:290]:
```suggestion
`loginEndpoint` | `string` | No | Endpoint for login, e.g. `/api/token`
```

File: README.md [hunk 4]
```diff
@@ -225,3 +236,148 @@ export class AuthEffects {
...
+`refreshTokenEndpoint` | `string` | No |
```
Comment (astorozhevsky) [inline README.md:291]:
```suggestion
`refreshTokenEndpoint` | `string` | No | Endpoint for refreshing token, e.g. `/api/token/refresh`
```

File: README.md [hunk 5]
```diff
@@ -225,3 +236,148 @@ export class AuthEffects {
...
+`refreshTokenEndpointMethod` | `'get' \| 'post'` | No |
```
Comment (astorozhevsky) [inline README.md:292]:
```suggestion
`refreshTokenEndpointMethod` | `'get' \| 'post'` | No | HTTP Method that will be used for calling endpoint to refresh token
```

**Review by astorozhevsky — CHANGES_REQUESTED** [review]

**Comment by RuslanAktaev** [comment]
@astorozhevsky fixed, check please

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+## Process
+
+1. Fork the project
+1. Create a new branch
+1. Code, test, commit and push
+1. Open a pull request detailing your changes. Make sure to follow the [template](.github/PULL_REQUEST_TEMPLATE.md)
+
+## Guidelines
+
+* Please follow the [TypeScript Style Guide by Google](https://google.github.io/styleguide/tsguide.html) and[Angular coding style guide](https://angular.io/guide/styleguide).
```
Comment (astorozhevsky) [inline CONTRIBUTING.md:15]:
```suggestion
* Please follow the [TypeScript Style Guide by Google](https://google.github.io/styleguide/tsguide.html) and [Angular coding style guide](https://angular.io/guide/styleguide).
```

**Review by astorozhevsky — CHANGES_REQUESTED** [review]

**Review by astorozhevsky — APPROVED** [review]

**Pull request merged**